src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -1571,13 +1290,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
         End Function
 
         Private Function ScanIdentifierOrKeyword(precedingTrivia As SyntaxList(Of VisualBasicSyntaxNode)) As SyntaxToken
-            Debug.Assert(CanGet)
-            Debug.Assert(IsIdentifierStartCharacter(Peek))
+            Dim ch As Char
+            Dim ok = TryPeek(ch)
```

Comment:
Actually here we use result of TryPeek call in production code. Please ignore previous comment.